### DISCUSSION TITLE
word of thanks

### DISCUSSION THREAD

**Discussion by allandf** [hawky358/hass_ids_hyyp#1] [General]
Hi there,

just wanted to say thanks for fixing this integration, greatly appreciated!

**Comment by e1ace** [comment]
I'm adding to this. A couple months ago I could never get the version working (some random error that I couldn't be bothered to debug). 

This worked without issues. Thanks for the effort.

**Comment by nico768** [comment]
Good job.
Just installed the HYYP IP module and the integrations to HA worked the first time.
Thank you !!
